File: problems/1925-count-square-sum-triples/solution.py
class Solution:
    def countTriples(self, n: int) -> int:
        count = 0
        for a in range(1, n + 1):
            for b in range(1, n + 1):
                c_squared = a * a + b * b
                c = int(c_squared**0.5)
                if c * c == c_squared and 1 <= c <= n:
                    count += 1
        return count

File: problems/1925-count-square-sum-triples/solution.java
class Solution {
    public int countTriples(int n) {
        int count = 0;
        for (int a = 1; a <= n; a++) {
            for (int b = 1; b <= n; b++) {
                int cSquared = a * a + b * b;
                int c = (int) Math.sqrt(cSquared);
                if (c * c == cSquared && c <= n) {
                    count++;
                }
            }
        }
        return count;
    }
}